src/main/java/org/commcare/formplayer/beans/menus/EntityListResponse.java

```diff
@@ -278,6 +280,27 @@ private static EntityBean processEntity(Detail detail, TreeReference treeReferen
         return ret;
     }
 
+    private static String processAutolaunch(SessionWrapper session) {
+        Vector<Action> actions = getActions(session);
+        String ret = null;
+        int index = 0;
+        for (Action action: actions) {
```

Comment:
is it possible for there to be 2 autolaunch actions? this looks like it would just return the second if thats the case. Is that ok?

Reply:
The XML spec does support multiple autolaunch actions, but they don't really make sense so I think this is fine. In retrospect, it might have made more sense to add an `autolaunch-action="0"` attribute to the detail instead of a flag on actions themselves. I'm not going to worry about that right now, but might think about it before removing the new feature flag.